students/slawinski_amadeusz/lesson_04_unit_testing/fibonacci.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+import unittest
+
+
+def fib(n):
+    if int(n) < 0 or isinstance(n, float):
```

Comment:
można użyć:
`if not isinstance(n, int) or n < 0:`
nie musimy tyle razy wykonywać int(n)